assets/sv.json

```diff
@@ -39,7 +39,18 @@
   "ETHAKU_HAE_TUNNUKSELLA": "Sök med certifikatkod",
   "ETHAKU_HAE_ALUEELLA": "Sök med område",
   "ETHAKU_HAE_ASIDE": "Du kan söka på landskap, kommun, postnummer eller postanstalt.",
-  "ETHAKU_ASIDE": "Ett sammandrag visas av energicertifikatet, dvs. uppgifterna på de två första sidorna (på grund av personskyddet dock inte när det gäller objekt med 1–2 lägenheter).",
+  "ETHAKU_ASIDE": "Ett sammandrag visas av energicertifikatet, dvs. uppgifterna på de två första sidorna (på grund av personskyddet dock inte när det gäller objekt med 1–2 lägenheter).",  "ETHAKU_INVALID_RAKENNUKSEN_NIMI": "Nimi voi olla korkeintaan 100 merkkiä",
+  "ETHAKU_INVALID_RAKENNUSTUNNUS": "Syötetty arvo ei ole oikeamuotoinen rakennustunnus",
+  "ETHAKU_INVALID_VUOSI_MIN": "Alkuvuoden tulee olla loppuvuotta pienempi",
+  "ETHAKU_INVALID_VUOSI_MAX": "Loppuvuoden tulee olla alkuvuotta suurempi",
+  "ETHAKU_INVALID_ALLEKIRJOITUSAIKA_MIN": "Alkupäivämäärän tulee olla ennen loppupäivämäärää",
+  "ETHAKU_INVALID_ALLEKIRJOITUSAIKA_MAX": "Loppupäivämäärän tulee olla alkupäivämäärän jälkeen",
+  "ETHAKU_INVALID_VOIMASSAOLO_MIN": "Alkupäivämäärän tulee olla ennen loppupäivämäärää",
+  "ETHAKU_INVALID_VOIMASSAOLO_MAX": "Loppupäivämäärän tulee olla alkupäivämäärän jälkeen",
+  "ETHAKU_INVALID_E_LUKU_MIN": "Arvon tulee olla loppuarvoa pienempi",
+  "ETHAKU_INVALID_E_LUKU_MAX": "Arvon tulee olla alkuarvoa suurempi",
+  "ETHAKU_INVALID_NETTOALA_MIN": "Arvon tulee olla loppuarvoa pienempi",
+  "ETHAKU_INVALID_NETTOALA_MAX": "Arvon tulee olla alkuarvoa suurempi",

```

Comment:
Nää pitäis tagata tai jättää laittamatta. Muuten menee helposti ohi käännöksessä.